docs/sources/alerting/fundamentals/alert-rules/annotation-label.md

```diff
@@ -28,12 +28,10 @@ Labels and annotations contain information about an alert. Labels are used to di
 
 When creating alert rules, you can also template labels and annotations to optimize and customize your alerts.
 
-**Labels** serve as unique identifiers of an alert. And, beyond their labeling function, they also offer practical functionalities such as searchability, silencing, routing notifications, and they can be templated.
-
-**Annotations** provide additional information in alert notification messages to help you understand the data that is being alerted on. For example, you can add a **Summary** annotation to tell you which value caused the alert rule to fire or which server it happened on. Annotations can contain a combination of text and template code.
-
 ## Labels
 
+**Labels** serve as unique identifiers of an alert. And, beyond their labeling function, they also offer practical functionalities such as searchability, silencing, routing notifications, and they can be templated.
+
 Labels contain information that identifies an alert. An example of a label might be `server=server1` or `team=backend`. Each alert can have more than one label, and the complete set of labels for an alert is called its label set. It is this label set that identifies the alert.
 
 For example, an alert might have the label set `{alertname="High CPU usage",server="server1"}` while another alert might have the label set `{alertname="High CPU usage",server="server2"}`. These are two separate alerts because although their `alertname` labels are the same, their `server` labels are different.
```

Comment:
```suggestion
For example, an alert rule might have the label set `{alertname="High CPU usage",server="server1"}` while another alert rule might have the label set `{alertname="High CPU usage",server="server2"}`. These are two separate alert rules because although their `alertname` labels are the same, their `server` labels are different.
```